from data.data import Datos
import custom_exceptions

class DatosRecomendacion(Datos):
    pass